src/float/float_transceiver/float_transceiver.ino

```diff
@@ -21,6 +24,10 @@ const uint8_t SUCK_PIN = 10;  // Set high for suck (CCW when facing down)
 const uint8_t LIMIT_FULL = 12;   // Low when syringe is full
 const uint8_t LIMIT_EMPTY = 11;  // Low when syringe is empty
 
+// Encoder initialization
+PicoEncoder encoder;
```

Comment:
Can this be const?

Reply:
No it cannot. Will move it down with the other drivers.